docs/building_blocks_for_flexible_content.md

```diff
@@ -15,7 +15,13 @@ blocks:
 - type: block_type
 ```
 
-This may include a top-level `navigation_groups` element for use by blocks (See [Navigation Groups](#navigation-groups)), or a top-level `extends` element to inherit configuration from another landing page (See [Extending a page (Whitehall Only)](#extending-a-page-whitehall-only)).
+It may optionally include other top-level elements:
+
+* a `breadcrumbs` element, an array of `title/href` pairs which will be used to create a breadcrumb list.
+* an `extends` element to inherit configuration from another landing page (See [Extending a page (Whitehall Only)](#extending-a-page-whitehall-only)).
+* an `image` element. This has one sub-element (`url`) which should be the url for an image to use with social media metadata.
+* a `navigation_groups` element for use by blocks (See [Navigation Groups](#navigation-groups))
+* a `theme` element which will set the general appearance of the page. Valid themes are `default` or `prime-ministers-office-10-downing-street`) - if no theme is specified, `default` will be used.
```

Comment:
This update to the documentation is great!

I think though that a section on themes is needed. The themes described here apply to the whole page, whereas there are themes that only apply to a block, e.g. hero blocks.

Reply:
Good call, done!